### PR TITLE
Add docs for Docker Azure integration beta

### PR DESCRIPTION
**Pull request by usha-mandya**
Docs for the Docker Azure integration beta release

### CODE REVIEW

**Comment by usha-mandya** [comment]
@nebuk89 @gtardif @chris-crone PTAL - https://deploy-preview-11036--docsdocker.netlify.app/engine/context/aci-integration/.

**Comment by usha-mandya** [comment]
Will add Desktop download links when ready.

File: engine/context/aci-integration.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+to a directory in your `PATH` with higher priority than the existing Docker CLI.
+
+Again, on a fresh Ubuntu 20.04:
+
+```console
+$ which docker
+/usr/bin/docker
+$ echo $PATH
+/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
+$ sudo mv docker-aci /usr/local/bin
```
Comment (chris-crone) [inline engine/context/aci-integration.md:217]: I fixed this elsewhere and didn't let you know, sorry.

```suggestion
$ sudo mv docker-aci /usr/local/bin/docker

```

File: engine/context/aci-integration.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+>
+> The current Docker Azure integration does not allow fetching a combined log stream from all the containers that make up the Compose application.
+
+## Using ACI resource groups as namespaces
+
+You can create several Docker contexts associated with ACI. Each context must be associated with a unique Azure resource group. This allows you to use Docker contexts as namespaces. You can switch between namespaces using `docker context use <CONTEXT>`.
+
+When you run the `docker ps` command, it only lists containers in your current Docker context. There won’t be any contention in container names or Compose application names between two Docker contexts.
+
+## Install the Docker ACI Integration CLI on Linux
```
Comment (chris-crone) [inline engine/context/aci-integration.md:135]: Maybe we should just point them to the GitHub repo for Linux? That way we don't have to maintain the docs in two places

File: engine/context/aci-integration.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+>**Note**
+>
+> Docker Azure Integration is currently a beta release. The commands and flags are subject to change in subsequent releases.
+{:.important}
+
+## Prerequisites
+
+To deploy Docker containers on Azure, you must meet the following requirements:
+
+1. Download and install Docker Desktop Edge version 2.3.2.0.
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply engine/context/aci-integration.md:28]: That's right. Waiting for a confirmation on the build. I'll update this when it's ready.

File: engine/context/aci-integration.md (hunk 1 above)
Comment (usha-mandya) [reply engine/context/aci-integration.md:217]: No worries. I'll update this.

File: engine/context/aci-integration.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+keywords: Docker, Azure, Integration, ACI, context, cli, deploy, containers, cloud
+toc_min: 1
+toc_max: 2
+---
+
+## Overview
+
+The Docker Azure Integration enables developers to use native Docker commands to run applications in Azure Container Instances (ACI) when building cloud-native applications. The new experience provides a tight integration between Docker Desktop and Microsoft Azure allowing developers to quickly run applications using the Docker CLI or VS Code extension, to switch seamlessly from local development to cloud deployment.
+
+In addition, the integration between Docker and Microsoft developer technologies offers the following benefits that enable developers to:
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply engine/context/aci-integration.md]: Fixed

File: engine/context/aci-integration.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+    - Download for Mac
+    - Download for Windows
+
+    Alternatively, install the [Docker ACI Integration for Linux](#install-the-docker-aci-integration-cli-on-linux).
+
+2. Ensure you have an Azure subscription. You can get started with an [Azure free account](https://aka.ms/AA8r2pj){: target="_blank" class="_"}.
+
+## Run Docker containers on ACI
+
+Docker not only runs containers locally, but also enables developers to seamlessly deploy Docker containers on ACI using docker run or deploy multi-container applications defined in a Compose file using the `docker compose up` command.
```
[the remark this answers is not in the document]
Comment (usha-mandya) [reply engine/context/aci-integration.md]: Fixed

File: engine/context/aci-integration.md (hunk 1 above)
Comment (usha-mandya) [reply engine/context/aci-integration.md]: Fixed.

File: engine/context/aci-integration.md (hunk 2 above)
Comment (usha-mandya) [reply engine/context/aci-integration.md:135]: Discussed with Chris.

File: engine/context/aci-integration.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (usha-mandya) [reply engine/context/aci-integration.md]: Updated.

**Comment by nebuk89** [comment]
LGTM

**Pull request merged**